src/components/Feedback/Feedback.js

```diff
@@ -8,7 +8,7 @@
  */
 
 import React from 'react';
-import withStyles from 'isomorphic-style-loader/lib/withStyles';
+import withStyles from 'isomorphic-style-loader/withStyles';
```

Comment:
This still isn't working with `yarn start` locally, see:

<img width="1270" alt="image" src="https://github.com/josephfrazier/reported-web/assets/6473925/3442ef98-17e7-4864-8b1f-3e53284ae1a5">

> # TypeError
> 
> ```
> TypeError: Cannot read properties of null (reading 'apply')
>     at new WithStyles (/Users/josephfrazier/workspace/reported-web/node_modules/isomorphic-style-loader/src/withStyles.js:21:26)
>     at processChild (/Users/josephfrazier/workspace/reported-web/node_modules/react-dom/cjs/react-dom-server.node.development.js:2995:14)
>     at resolve (/Users/josephfrazier/workspace/reported-web/node_modules/react-dom/cjs/react-dom-server.node.development.js:2960:5)
>     at ReactDOMServerRenderer.render (/Users/josephfrazier/workspace/reported-web/node_modules/react-dom/cjs/react-dom-server.node.development.js:3435:22)
>     at ReactDOMServerRenderer.read (/Users/josephfrazier/workspace/reported-web/node_modules/react-dom/cjs/react-dom-server.node.development.js:3373:29)
>     at Object.renderToString (/Users/josephfrazier/workspace/reported-web/node_modules/react-dom/cjs/react-dom-server.node.development.js:3988:27)
>     at /Users/josephfrazier/workspace/reported-web/src/server.js:632:1
>     at processTicksAndRejections (node:internal/process/task_queues:95:5)
> ```
>